administration/src/util/generateDeepLink.ts

```diff
@@ -0,0 +1,15 @@
+import { ACTIVATION_FRAGMENT, ACTIVATION_PATH } from 'build-configs/constants'
+
+import { PdfQrCode } from '../cards/pdf/PdfQrCodeElement'
+import { QrCode } from '../generated/card_pb'
+import { uint8ArrayToBase64 } from './base64'
+
+const generateDeepLink = (qrCode: PdfQrCode): string => {
+  const qrCodeContent = new QrCode({
+    qrCode: qrCode,
+  }).toBinary()
+  return `${window.location.origin}/${ACTIVATION_PATH}/${ACTIVATION_FRAGMENT}${encodeURIComponent(
```

Comment:
the host and protocol should come from the project config, so that I can use this url during development.
-> `berechtigingskarte://bayern.ehrenamtskarte.app/...`

Reply:
Now i'm facing compile errors @sarahsporck when i'm using the `getBuildConfig()`

Reply:
I only get it work with
https://github.com/digitalfabrik/entitlementcard/pull/1328